source/index.html.md

```diff
@@ -0,0 +1,429 @@
+
+
+
+<aside class="notice">
+Make sure you are a Turntabl employee before assess this API
+</aside>
+
+# Project  Controller
+
+## Get All Projects
+
+
+```shell
+curl "https://project.services.turntabl.io/v1/api/projects" 
+-H  "accept: */*"  
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+
+  {
+    "project_id": 1,
+    "project_name": "TPMS WEB APP"    
+  },
+  {
+   "project_id": 2,
+    "project_name": "AWS"   
+  }
+
+```
+
+This endpoint is used to retrieve all projects.
+
+### HTTP Request
+
+
+`GET https://project.services.turntabl.io/v1/api/projects`
+
+
+
+## Add New Project
+
+
+```shell
+curl "https://project.services.turntabl.io/v1/api/project" 
+-H  "accept: */*" 
+-H  "Content-Type: application/json" 
+-d "{  \"project_name\": \"string\"}"
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+
+  {   
+    "project_name": "string"    
+  }
+
+```
+
+This endpoint is used to add new project.
+
+### HTTP Request
+
+
+`POST https://project.services.turntabl.io/v1/api/project`
+
+
+
+
+
+
+
+## Get Projects Assigned to Employee By His/Her ID
+
+
+```shell
+curl "https://project.services.turntabl.io/v1/api/projects/assigned/employee/id" 
+-H  "accept: */*"
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "id": 2,
+  "project_name": "AWS",
+  
+}
+```
+
+This endpoint is used to retrieve a specific employee's project.
+
+
+### HTTP Request
+
+`GET https://project.services.turntabl.io/v1/api/projects/assigned/employee/id<ID>`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+ID | The ID of the Employee to retrieve his/her project
+
+
+
+
+
+## Assigned Project to Employee
+
+
+```shell
+curl -X POST "https://project.services.turntabl.io/v1/api/project/assign/employee" 
+-H  "accept: */*" 
+-H  "Content-Type: application/json" 
+-d "{  \"employee_email\": \"string\",  \"employee_firstname\": \"string\",  \"employee_id\": 0,  \"employee_lastname\": \"string\",  \"project_id\": 0}"
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "employee_email": "francis.billa@turntabl.io",
+  "employee_firstname": "francis",
+  "employee_id": 4,
+  "employee_lastname": "billa",
+  "project_id": 2
+}
+```
+
+This endpoint is used to assign a project to a specific employee
+
+
+### HTTP Request
+
+`POST https://project.services.turntabl.io/v1/api/project/assign/employee<ID>`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+ID | The ID of the Employee to retrieve his/her project
+
+
+# Logged Hours Controller
+
+## Get All Logged Hours
+
+
+```shell
+curl -X GET "https://project.services.turntabl.io/v1/api/{end_date}" 
+-H  "accept: */*"
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+
+  {
+    "project_id": 1,
+    "project_name": "TPMS WEB APP"    
+  }
+
+```
+
+This endpoint to retrieves all logged project hours.
```

Comment:
This endpoint retrieves all the hours logged against all projects

Reply:
done